app/src/test/java/com/github/multimatum_team/multimatum/GroupAdapterTest.kt

```diff
@@ -10,15 +10,10 @@ import androidx.test.espresso.intent.Intents
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.github.multimatum_team.multimatum.adaptater.UserGroupAdapter
 import com.github.multimatum_team.multimatum.model.*
-import com.github.multimatum_team.multimatum.repository.AuthRepository
-import com.github.multimatum_team.multimatum.repository.DeadlineRepository
-import com.github.multimatum_team.multimatum.repository.GroupRepository
-import com.github.multimatum_team.multimatum.repository.UserRepository
-import com.github.multimatum_team.multimatum.util.MockAuthRepository
-import com.github.multimatum_team.multimatum.util.MockDeadlineRepository
-import com.github.multimatum_team.multimatum.util.MockGroupRepository
-import com.github.multimatum_team.multimatum.util.MockUserRepository
+import com.github.multimatum_team.multimatum.repository.*
+import com.github.multimatum_team.multimatum.util.*
 import com.github.multimatum_team.multimatum.viewmodel.GroupViewModel
+import com.google.firebase.storage.FirebaseStorage
```

Comment:
No test other than `FirebaseXRepositoryTest` should (and they mock firebase behavior anyway so no direct call to firebase is done)